diagrams/__init__.py

```diff
@@ -87,6 +88,9 @@ def __init__(
         graph_attr: dict = {},
         node_attr: dict = {},
         edge_attr: dict = {},
+        engine: str = "dot",
+        nodesep: str = "0.60",
+        ranksep: str = "0.75",
```

Comment:
`nodesep` and `ranksep` can be passed with `graph_attr` parameter. I think we don't have to receive these attributes individually.